### DISCUSSION TITLE
Do you have documentaion with examples of files?

### DISCUSSION THREAD

**Discussion by anvme** [spacecatninja/craft-imager-x#225] [General]
There is no information with real examples of how all this works.
It's impossible to understand with real models. Only shortcake for specific cases. 

Usually plugins with code, no GUI for $100 🤯 have some real examples with full code for all config files.

**Comment by aelvan** [comment]
No, I don't see the point of maintaining an example config file with all config settings. Neither does Pixel&Tonic, there's not one for Craft itself.
